gocd/templates/pipelines/objectstore.libsonnet

```diff
@@ -1,43 +1,51 @@
 local utils = import '../libs/utils.libsonnet';
 local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/libs/gocd-tasks.libsonnet';
 
-local deploy_canary(region) = [
-  {
-    'deploy-canary': {
-      fetch_materials: true,
-      jobs: {
-        create_sentry_release: {
-          environment_variables: {
-            SENTRY_ORG: 'sentry',
-            SENTRY_PROJECT: 'objectstore',
-            SENTRY_AUTH_TOKEN: '{{SECRET:[devinfra-sentryio][token]}}',
-            SENTRY_ENVIRONMENT: region + '-canary',
-          },
-          timeout: 60,
-          elastic_profile_id: 'objectstore',
-          tasks: [
-            gocdtasks.script(importstr '../bash/create-sentry-release.sh'),
-          ],
-        },
-        deploy: {
-          timeout: 300,
-          elastic_profile_id: 'objectstore',
-          environment_variables: {
-            K8S_ENVIRONMENT: 'canary',
-            PAUSE_MESSAGE: 'Pausing pipeline due to canary failure.',
+// NOTE: Sync with objectstore-k8s in getsentry/ops.
+local region_has_canary(region) =
+  region == 'de' || region == 'us';
+
+local deploy_canary(region) =
+  if region_has_canary(region) then
+    [
```

Comment:
All of this was indented by 2 levels, no changes to the canary stage internally.